docs/design/2021-03-09-security-enhanced-mode.md

```diff
@@ -0,0 +1,191 @@
+# Proposal:
```

Comment:
Well, sorry for the inconveniences but we've updated the template of the design doc(https://github.com/pingcap/tidb/blob/master/docs/design/TEMPLATE.md), please update your PR accordingly.